app/dose-series-slider/dose-series-slider.component.ts

```diff
@@ -35,5 +35,18 @@ export class DoseSeriesSliderComponent {
             }
         );
     }
+
+    slidingEdit(slidingItem: ItemSliding) {
+        //redirct to the edit screen
+        this.doseSeriesService.edit(this.doseSeries).then(
```

Comment:
All this does is call the PUT method on the server.  This isn't the place for that.  Instead, we need to either refactor the new-dose-medication page to be able to take in a dose-series as input and populate that form and switch between calling edit and save on the service, or create a new page that looks exactly the same but can take in a dose series as input.

I was looking at the fabs for an example, so this shouldn't be difficult.